geoportal/geoportailv3_geoportal/views/authentication.py

```diff
@@ -88,9 +88,9 @@ class O(object):
                 if 'roleOGC' in obj:
                     user.ogc_role = int(obj['roleOGC'][0])
         try:
-            user.role = DBSession.query(Role).filter_by(id=user.mymaps_role).one()
+            user.role = DBSession.query(Role).filter_by(id=roletheme).one()
         except Exception as e:
-            user.role = DBSession.query(Role).filter_by(id=default_mymaps_role).one()
```

Comment:
@rmichaelis, why don't you define the DEFAULT_MYMAPS_ROLE environment variable?

Reply:
In my database I only have 1 role: role_admin with id 645.
Would you be able to include the `role` table in the dump?

Having a few test maps and associated test users would be great too.
Don't you have some test users for the production db?

Reply:
@gberaudo The issue GSLUX-157 already contains a full export of our production database. (including role table)
Don't you work with this dump ?

Reply:
> @rmichaelis, why don't you define the DEFAULT_MYMAPS_ROLE environment variable?

DEFAULT_MYMAPS_ROLE is dedicated to mymaps and not to themes. I could use an env variable such as DEFAULT_ROLE_THEME instead.

Reply:
@rmichaelis, I will fix our luxembourg-dev-db values so that role_admin has id 0.
So the change LGTM.

Reply:
@gberaudo Or assign the role 645 to your test user.

Reply:
Hmm, I see, I have both a role and a geov3.role table. I will check what happens.